state/workers.go

```diff
@@ -53,20 +59,14 @@ func newWorkers(st *State, hub *pubsub.SimpleHub) (*workers, error) {
 			Clock:        st.clock(),
 		}),
 	}
-	if hub == nil {
-		ws.StartWorker(txnLogWorker, func() (worker.Worker, error) {
-			return watcher.New(st.getTxnLogCollection()), nil
+	ws.StartWorker(txnLogWorker, func() (worker.Worker, error) {
```

Comment:
Do we care that none of these `StartWorker` calls are error checked?

Reply:
A little, because it has been the cause of a few hard to determine bugs. Should get to it sometime.